ndmg/register.py

```diff
@@ -266,6 +267,17 @@ def gen_tissue(self):
         print("Extracting brain from raw T1w image...")
         reg_utils.t1w_skullstrip(self.t1w, self.t1w_brain, self.skull)
 
+        #  QA part of skull strip
+        last_folder = self.namer.dirs["qa"]["base"]
+        QA_skullstrip_path = last_folder + '/skull_strip'
+        folder = os.path.exists(QA_skullstrip_path)
+        if not folder:
```

Comment:
change to `if not os.path.exists(self.namer.dirs["qa"]["skull_strip"]:`
`os.makedirs(self.namer.dirs["qa"]["skull_strip"]`

Reply:
fixed it in latest version

Reply:
don't say "`fixed it in latest version`", it makes it hard to keep track of what you've done.
instead, comment with the commit that you fixed it in,
and then click `Resolve Conversation`.

Reply:
See this as an example: https://github.com/scipy/scipy/pull/10524#discussion_r307915803

If you have questions/comments, do that as well of course.

Reply:
![image](https://user-images.githubusercontent.com/55203069/70834643-e1f7ce80-1dc8-11ea-80b6-39c1af1d05e5.png)